staging/src/k8s.io/kubectl/pkg/util/openapi/dryrun_test.go

```diff
@@ -1,80 +0,0 @@
-/*
```

Comment:
Where is this gone? :-o

Reply:
Oh, this test is merged into `staging/src/k8s.io/cli-runtime/pkg/resource/dry_run_verifier_test.go`.

I rebased to make sure git sees the mv, but GitHub still doesn't see it.